src/Core/Makefile.am

```diff
@@ -7,3 +13,10 @@ SUBDIRS = \
 		FSpot.Query \
 		FSpot.Gui \
 		FSpot.Platform
+
+RESOURCES = \
+		FSpot.Core.UnitTest/TestData/taglib-sample.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample.jpg \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.jpg
+		
```

Comment:
I agree they are needed.  Even when installing from tarball, it is good to be able to run the tests as an additional sanity check of your environment.  I think this is particularly true for source based distributions such as Gentoo or LFS.